sdk/python/pulumi_kubernetes/__init__.py

```diff
@@ -1,39 +1,40 @@
 # coding=utf-8
-# *** WARNING: this file was generated by the Pulumi Kubernetes codegen tool. ***
+# *** WARNING: this file was generated by pulumigen. ***
 # *** Do not edit by hand unless you're certain you know what you are doing! ***
 
+import importlib
+# Export this package's modules as members:
+from .provider import *
+from .yaml import *
```

Comment:
Does this expose the internal methods as well? `ConfigFile` was the only user-facing function here IIRC.

Reply:
So the difference here is that before `from pulumi_kubernetes import _parse_yaml_document` would be an error, but now it's only a warning. But even in the previous case you could always do `from pulumi_kubernetes.yaml import _parse_yaml_document` which would warn but allow you to import the "private" method.

The way to maintain the existing functionality would be to add `__all__ = ['ConfigFile']` at the top of yaml.py

Reply:
https://github.com/pulumi/pulumi-kubernetes/pull/1160/files#diff-a5a9a736b71d7d0aa6ed60512611a75bR15